content/docs/releases/enterprise/install/quickstart.mdx

```diff
@@ -12,218 +12,278 @@ import GenerateRecoveryToken from '@site/content/_generate-recovery-token.md';
 
 # Pomerium Enterprise Quickstart
 
-## Before You Begin
+Learn how to run Pomerium Enterprise using Docker containers.
 
-This document assumes:
+## Prerequisites
 
-- A non-containerized environment, either your local computer or a virtual machine (**vm**). While Pomerium is designed to scale with your production environment, we'll leave containerization and infrastructure as code (**IaC**) out for now, to focus on learning how Pomerium Enterprise works.
-  - `root` or `sudo` privileges on the host.
-- You already have the open-source Pomerium base installed. If not, follow [this doc](/docs/releases/core) before you continue.
-  - While an existing route is not required, we suggest implementing one test route to validate your identity provider (**IdP**) configuration.
-- Pomerium Enterprise requires a relational database. PostgreSQL 9+ is supported.
-  - Securing the database connection with TLS may not be required, especially for a local installation, but is strongly recommended for production deployments. Therefore, this guide will assume a TLS-secured database connection.
+To complete this guide, you need:
 
-## Requirements
+- Pomerium Core
+- A configured identity provider
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- PostgreSQL
+- Prometheus
 
-- Pomerium Enterprise requires Linux amd64/x86_64. It can manage Pomerium instances on other platforms, however.
-- Each Console instance should have at least:
-  - 4 vCPUs
-  - 8G RAM
-  - 100G of disk wherever logs are stored
-- Each Postgres instance should have at least:
-  - 4 vCPUs
-  - 8G RAM
-  - 20G for data files
+This guide runs Pomerium Core and PostgreSQL as Docker containers. If you haven't, pull the [Pomerium](https://hub.docker.com/r/pomerium/pomerium) and [PostgreSQL](https://hub.docker.com/_/postgres) Docker images from Docker Hub and complete the [Pomerium Core quickstart](/docs/quickstart) to set up a Pomerium instance that's compatible with this guide.
 
-## Install Pomerium Enterprise
+The Pomerium Enterprise Console uses an embedded Prometheus server to collect and display metrics. While you can [configure metrics](/docs/capabilities/metrics) to use an existing Prometheus server, this guide uses Pomerium’s Prometheus backend.
 
-Pomerium publishes standard OS packages for RPM and DEB based systems. The repositories require authentication via username and access key. These credentials will be issued to you during the onboarding process.
+:::tip **Note**
 
-<Tabs>
+This guide assumes you've already registered for Pomerium Enterprise and have credentials to access the private Cloudsmith Docker registry. If you haven't registered for Pomerium Enterprise, [sign up here](https://www.pomerium.com/enterprise-sales/) for a free trial.
```

Comment:
Suggestion:
~~credentials to access the private Cloudsmith Docker registry.~~
credentials to access the private Cloudsmith.io Docker registry, and a license key